capabilities/writetarget/beholder/monitor/metrics.go

```diff
@@ -0,0 +1,48 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder/utils"
+)
+
+// Define a new gauge metric for account balance
+type GaugeAccBalance struct {
+	// account_balance
+	gauge metric.Float64Gauge
+}
+
+func NewGaugeAccBalance(unitStr string) (*GaugeAccBalance, error) {
```

Comment:
This is something that is not directly tied to WT capability, and is a separate framework to construct a balance monitor. It should not be grouped under `/writetarget/...`

It potentially belongs in this repo, on another path.

Reply:
Added TODO [here](https://github.com/smartcontractkit/chainlink-framework/blob/e363918a2b6b6f9cec000247ec636e42ccb66efe/capabilities/writetarget/beholder/monitor/metrics.go#L15)